include/nncase/ir/op_utils.h

```diff
@@ -127,6 +127,11 @@ inline shape_t normalize_reshape(const shape_t &in_shape, const axis_t &new_shap
                 throw std::runtime_error("Reshap can only have 1 non-determined dimension at most");
             non_det_id = i;
         }
+        else if(v == 0)
```

Comment:
What's the basis of this processing?

Reply:
https://caffe.berkeleyvision.org/tutorial/layers/reshape.html
“0 means “copy the respective dimension of the bottom layer”. That is, if the bottom has 2 as its 1st dimension, the top will have 2 as its 1st dimension as well, given dim: 0 as the 1st target dimension.”

Reply:
It seems a Caffe-specific definition. I think it's better not to put it in the general purpose `op_utils.h`.

Reply:
Yes, but there is not a simple way to modify since it's call by IR. If you want to distinguish this, IR should be changed. Suggest don't do big modification currently because there is no ambiguity among frameworks we support now.

Reply:
You can normalize the shape at import stage.

Reply:
Ok, done.